src/dbus/server/dbus_request.hpp

```diff
@@ -162,6 +165,19 @@ class DBusRequest
             reply = UniqueDBusMessage(dbus_message_new_error(mMessage, ConvertToDBusErrorName(aError), nullptr));
         }
 
+        if (aResult.HasValue())
+        {
+            DBusMessageIter replyIter;
+            otbrError       error;
+
+            dbus_message_iter_init_append(reply.get(), &replyIter);
+            error = DBusMessageEncode(&replyIter, *aResult);
```

Comment:
Should we die on such error ?

Reply:
It should die, I think. But it doesn't die on error of `dbus_message_new_method_return`. Anyway, I made both cases die.